e2e/cypress/integration/team/teams_spec.js

```diff
@@ -182,4 +182,50 @@ describe('Teams Suite', () => {
         // * Ensure user is logged out
         cy.url({timeout: TIMEOUTS.LARGE}).should('include', 'login');
     });
+
+    it('TS14632 Team Icon should be Removed', () => {
+        cy.apiLogin('sysadmin');
+        cy.apiCreateTeam('test-team', 'Test Team').then(() => {
+            cy.visit('/');
+        });
+        cy.get('#sidebarHeaderDropdownButton').click();
+        cy.get('#teamSettings').click();
+
+        // 1. On a Team taht has a team icon
+        cy.get('#team_iconEdit').click();
+        cy.fileUpload('.section-max input', '../fixtures/mattermost-icon.png', 'image/png').trigger('change', {force: true});
+        cy.get('#saveSettingPicture').should('not.have.css', 'disabled');
+        cy.get('#saveSettingPicture').click();
+        cy.get('#teamSettingsModalLabel > .close').click();
+
+        // Making sure there is a team icon for active team
+        cy.get('.TeamIcon__image').should('be.visible');
+
+        // 2. Go to main menu > Team Settings > Team Icon > Edit
+        cy.get('#sidebarHeaderDropdownButton').click();
+        cy.get('#teamSettings').click();
+        cy.get('#team_iconEdit').click();
+
+        // 3. Click the x in the top right corner of the image, then click "Cancel".
+
+        cy.get('.team-img__remove').click();
```

Comment:
Please add ID as `teamImageRemove` so this can be:
```suggestion
        cy.get('#teamImageRemove').click();
```

Reply:
Same question as above